### PR TITLE
add launch mp distributed job

### PR DESCRIPTION
**Pull request by Yancey0623**
image

### CODE REVIEW

File: doc/fluid/user_guides/howto/training/cluster_howto.rst [hunk 1]
```diff
@@ -218,6 +218,27 @@ NCCL2模式的分布式训练，由于没有parameter server角色，是trainer
...
++++++++++++++++++++++++++++++++++
+
+通常情况下使用多进程模式启动 NCCL2 分布式训练作业可以获得更好多训练性能，Paddle 提供了
+:code:`paddle.distributed.launch` 模块可以方便地启动多进程作业，启动后每个训练进程将会使用一块独立的 GPU 设备。
+使用时需要注意：
+
+* 设置节点数：通过环境变量 :code:`PADDLE_NUM_TRAINERS` 设置作业地节点数，此环境变量也会被设置在每个训练进程中。
+* 设置每个节点的设备数：通过启动参数 :code:`--gpus` 可以设置每个节点的 GPU 设备数量，每个进程的序号将会被自动设置在环境变量
+  :code:`PADDLE_TRAINER_ID` 中。
+* 数据切分： 多进程模式是每个设备一个进程，一般来说需要每个每个进程处理一部分训练数据，并且保证所有进程能够处理完整的数据集。
```
Comment (typhoonzero) [inline doc/fluid/user_guides/howto/training/cluster_howto.rst]: 每个每个进程

Comment (Yancey0623) [reply doc/fluid/user_guides/howto/training/cluster_howto.rst]: thanks, done.

**Review by typhoonzero — COMMENTED** [review]
LGTM, please @shanyi15 take a look at this

File: doc/fluid/user_guides/howto/training/cluster_howto.rst [hunk 2]
```diff
@@ -218,6 +218,27 @@ NCCL2模式的分布式训练，由于没有parameter server角色，是trainer
...
+
+通常情况下使用多进程模式启动 NCCL2 分布式训练作业可以获得更好多训练性能，Paddle 提供了
+:code:`paddle.distributed.launch` 模块可以方便地启动多进程作业，启动后每个训练进程将会使用一块独立的 GPU 设备。
+使用时需要注意：
+
+* 设置节点数：通过环境变量 :code:`PADDLE_NUM_TRAINERS` 设置作业地节点数，此环境变量也会被设置在每个训练进程中。
+* 设置每个节点的设备数：通过启动参数 :code:`--gpus` 可以设置每个节点的 GPU 设备数量，每个进程的序号将会被自动设置在环境变量
+  :code:`PADDLE_TRAINER_ID` 中。
+* 数据切分： 多进程模式是每个设备一个进程，一般来说需要每个进程处理一部分训练数据，并且保证所有进程能够处理完整的数据集。
+* 入口文件：入口文件为实际启动的训练脚本.
```
Comment (shanyi15) [inline doc/fluid/user_guides/howto/training/cluster_howto.rst]: 句末请统一用中文句号

Comment (Yancey0623) [reply doc/fluid/user_guides/howto/training/cluster_howto.rst]: thx, done.

**Review by shanyi15 — APPROVED** [review]
LGTM

**Comment by shanyi15** [comment]
@haowang101779990  new doc, please add to "to do list" of translation

**Pull request merged**